problems/README.md

```diff
@@ -1,12 +1,21 @@
 # Problems
 
-This folder contains all the initial files for the workshop.
+## Pre-requisites
+
+In order to run the problems, make sure you did the setup steps in the [main README](../README.md).
```

Comment:
This was drilled into my brain when I used to do technical writing: "In order to" and "be able to" are just empty words.

```suggestion
To run the problems, make sure you did the setup steps in the [main README](../README.md).